ttl/hbp_taxonomy_ontology.ttl

```diff
@@ -30,6 +30,7 @@ obo:NCBITaxon_10090 rdf:type owl:Class ;
                        rdfs:label "Mus musculus"@en ;
                        rdfs:subClassOf obo:NCBITaxon_9989 ;
                        HBP_PROP:has_rank HBP_TAX:9999996 ;
+                       HBP_PROP:has_brain_atlas_ontology HBP_MBA;
```

Comment:
I will delete those from the taxonomy ontology

From: Mohameth François SY <notifications@github.com<mailto:notifications@github.com>>
Reply-To: NeuroscienceKnowledgeSpace/methodsOntology <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday 11 April 2016 at 09:31
To: NeuroscienceKnowledgeSpace/methodsOntology <methodsOntology@noreply.github.com<mailto:methodsOntology@noreply.github.com>>
Cc: Silvia Jimenez <silvia.jimenez@epfl.ch<mailto:silvia.jimenez@epfl.ch>>
Subject: Re: [NeuroscienceKnowledgeSpace/methodsOntology] Added data types '.cef' and '.xlsx' for transcriptomics, updated data modalities (#33)


In ttl/hbp_taxonomy_ontology.ttl<https://github.com/NeuroscienceKnowledgeSpace/methodsOntology/pull/33#discussion_r59163569>:

> @@ -30,6 +30,7 @@ obo:NCBITaxon_10090 rdf:type owl:Class ;
>                         rdfs:label "Mus musculus"@en ;
>                         rdfs:subClassOf obo:NCBITaxon_9989 ;
>                         HBP_PROP:has_rank HBP_TAX:9999996 ;
> +                       HBP_PROP:has_brain_atlas_ontology HBP_MBA;


Is the property HBP_PROP:has_brain_atlas_ontology defined in this file or in /ttl/hbp_properties.ttl ?
If not, it should be defined in /ttl/hbp_properties.ttl.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/NeuroscienceKnowledgeSpace/methodsOntology/pull/33/files/a70722d8e8e65d9cd65f73a58e01d67016fac436#r59163569>